angular-app/src/app/app.component.html

```diff
@@ -1 +1 @@
-<router-outlet></router-outlet>
\ No newline at end of file
+<app-todo-shell></app-todo-shell>
```

Comment:
Ideea app-shell-ului este de a avea un container, scrollable, pentru toata aplicatia, cu elemente ce se pot folosi pentru toate paginiile actuale sau din viitor. In cazul nostru, ar fi doar header-ul, singurul element ce putem fi sigur ca o sa-l folosim si in alte pagini.